packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -493,6 +497,32 @@ describe('Ceramic interop: core <> http-client', () => {
     expect(doc.state.anchorStatus).toEqual(AnchorStatus.ANCHORED)
   })
 
+  it.only('feed works via http api', async () => {
+    let messageEvent: any
+    const Codec = JsonAsString.pipe(AggregationDocument)
+    const source = new EventSource(
+      `http://localhost:${daemon.port}/api/v0/feed/aggregation/documents`
+    )
+    source.addEventListener('message', (event) => {
+      messageEvent = decode(Codec, event.data)
+    })
+    //this will trigger a feed event
+    const MODEL_DEFINITION: ModelDefinition = {
+      name: 'myModel',
+      version: '1.0',
+      schema: { type: 'object', additionalProperties: false },
+      accountRelation: { type: 'list' },
+    }
+    // create model on different node
```

Comment:
is this actually being created on a different node?  Looks like this is the same node as we're getting the feed from

Reply:
you are right, the test was initially different and when I changed it didnt delete the comment. Thanks for catching this, just removed it.